src/webgl.cc

```diff
@@ -1764,11 +1764,11 @@ NAN_METHOD(GetExtension) {
   char *sname=*name;
   char *extensions=(char*) glGetString(GL_EXTENSIONS);
   char *ext=strcasestr(extensions, sname);
-  
-  if(ext==NULL){ 
+
+  if(ext==NULL){
```

Comment:
please just run clang-format on this file ☠️ ☠️ ☠️

Reply:
Ugh, I was thinking of the opposite: backing off the whitespace changes. Probably want a minimal change in case we want to upstream?

Reply:
I agree - in this case might it make sense to drop the whitespace changes